Documentation/REST API.md

```diff
@@ -485,3 +542,87 @@ uploadcare.videoConversionJobStatus(token: 123456) { result in
     }
 }
 ```
+
+
+## Add-Ons ([API Reference](https://uploadcare.com/api-refs/rest-api/v0.7.0/#tag/Add-Ons)) ##
+An Add-On is an application implemented by Uploadcare that accepts uploaded files as an input and can produce other files and/or appdata as an output.
+
+### Execute AWS Rekognition
```

Comment:
Maybe "// execute" should go inside the example, and title should be just the name of the addon?